### ISSUE TITLE
Support jsonlz4 as used in Firefox?

### ISSUE THREAD

**Issue by ur5us** [ouch-org/ouch#171]
Firefox uses lz4 compression with an offset (header) for their jsonlz4 files. As that something that could be added to ouch? Or would a dedicated tool or a fork be a better place as it’s too niche?

**Comment by marcospb19** [comment]
I don't think this would be possible just because our dependencies do not support this and we don't implement logic for each format, mainly because it requires much time and effort. Even if you added this to an existing lz4 library, and `ouch` could use it, I'm not sure if it would work.

If the library would automatically detected it while decompressing, ok, but for compressing, we would need to add a new flag, right?

But I think this is too niche to add a new flag, also because we try to keep `ouch` with a small --help message, I find other tools to have too many options and this makes it harder to learn and use.

But maybe I understood it wrong and it does not require any additional flag?
